roles/common/files/libexec/smart.sh

```diff
@@ -143,7 +143,8 @@ areca_smart ()
         messages+=("Drive $slot $dl has $reallocated reallocated sectors")
         failed=true
         # A small number of reallocated sectors is OK
-        if [ "$reallocated" -le 5 ]
+	# Don't set rc to WARN if we were already CRIT from previous drive
+        if [ "$reallocated" -le 5 ] && [ "$rc" != 2 ]
```

Comment:
logic starts being icky.  If we have to touch this again, might consider something like calling a "increase()" function to only increase rc, so you don't need if[] everywhere, or early-out from the tests if you have any fails, or something